src/helper/remote_config/client.hpp

```diff
@@ -0,0 +1,71 @@
+// Unless explicitly stated otherwise all files in this repository are
+// dual-licensed under the Apache-2.0 License or BSD-3-Clause License.
+//
+// This product includes software developed at Datadog
+// (https://www.datadoghq.com/). Copyright 2021 Datadog, Inc.
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include "http_api.hpp"
+#include "product.hpp"
+#include "protocol/client.hpp"
+#include "protocol/tuf/get_configs_response.hpp"
+
+namespace dds::remote_config {
+
+struct config_path {
+public:
+    config_path() = default;
+    std::string get_id() { return _id; };
```

Comment:
```suggestion
    std::string get_id() const{ return _id; };
```

You're returning a copy, perhaps a const reference is enough.

Reply:
Done in all getters